kibana/values.yaml

```diff
@@ -31,6 +31,13 @@ resources:
     cpu: "1000m"
     memory: "1Gi"
 
+# If Pod Security Policy in use it may be required to specify security context as well as service account
+securityContext:
```

Comment:
Can you rename this to podSecurityContext. Otherwise it will be a bit confusing if someone has a need to add container level security contexts too. https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod

Reply:
agree, done